app/controllers/purchases_controller.rb

```diff
@@ -43,7 +48,7 @@ def create_iugu_charge(iugu, course)
     response = iugu.charge.create(
```

Comment:
If I removed `iugu` from the parameters, the `create_iugu_charge` method complained about `AbcSize` o.O So I left it here even though it wasn't needed.